Packs/MicrosoftGraphSecurity/ReleaseNotes/2_2_12.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### Microsoft Graph Security
+Documentation and metadata improvements.
```

Comment:
Is this enough detail? Do you need to mention anything about improved descriptions for eDiscovery and Threat Assessment commands?